src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/resettlementassessmentstrategies/YamlResettlementAssessmentStrategy.kt

```diff
@@ -30,6 +31,23 @@ import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.jpa.repository.
 import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.service.getClaimFromJWTToken
 import java.time.LocalDateTime
 
+@JsonFormat(shape = JsonFormat.Shape.OBJECT)
+enum class GenericResettlementAssessmentQuestion(
+  override val id: String,
+  override val title: String,
+  override val subTitle: String? = null,
+  override val type: TypeOfQuestion,
+  override val options: List<Option>? = null,
+  override val validationType: ValidationType = ValidationType.MANDATORY,
+) : IResettlementAssessmentQuestion {
+  CASE_NOTE_SUMMARY(
+    id = "CASE_NOTE_SUMMARY",
+    title = "Add a case note summary",
+    subTitle = "This will be displayed as a case note in both DPS and nDelius",
+    type = TypeOfQuestion.LONG_TEXT,
+  ),
+}
+
 @Service
 class YamlResettlementAssessmentStrategy(
```

Comment:
We should be able to remove the interface `IResettlementAssessmentStrategy` and use this bean directly rather than using the `appliesTo` function